cmd/cloud.go

```diff
@@ -206,11 +202,12 @@ This will execute the test on the k6 cloud service. Use "k6 login cloud" to auth
 					}
 					globalCancel()
 				}()
-
-				sig = <-sigC
+			}
+			hardStop := func(sig os.Signal) {
 				logger.WithField("sig", sig).Error("Aborting k6 in response to signal, we won't wait for the test to end.")
-				os.Exit(int(exitcodes.ExternalAbort))
-			}()
```

Comment:
Or maybe just have `os.Exit` in both hardStops :shrug: I kind of feel like this DRY-ing of the code isn't really worth it

Reply:
:+1: for `gracefulStop` and `onHardStop`
Regarding the DRY, keep in mind that this will be used at least 2 more times, in `k6 coordinate` and `k6 agent`

Reply:
(for expediency, I will address this in the next PR)